client/scss/components/_opening_hours.scss

```diff
@@ -59,12 +59,15 @@
   @include font('HNL6');
   color: color('accessible-grey-2');
   width: 100%;
-  display: flex;
+  display: table;
```

Comment:
Does this tell screen readers to read it as table data?
And does setting it as flex do the opposite?

Why isn't this table by default as it's tables?